src/test/java/net/sf/jabref/importer/fileformat/OvidImporterTest.java

```diff
@@ -0,0 +1,143 @@
+package net.sf.jabref.importer.fileformat;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.bibtex.BibtexEntryAssert;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+
+public class OvidImporterTest {
+
+    private OvidImporter importer;
+
+
+    @Before
+    public void setUp() throws Exception {
+        if (Globals.prefs == null) {
+            Globals.prefs = JabRefPreferences.getInstance();
+        }
+        importer = new OvidImporter();
+    }
+
+    @Test
+    public void testGetFormatName() {
+        Assert.assertEquals("Ovid", importer.getFormatName());
+    }
+
+    @Test
+    public void testGetCLIId() {
+        Assert.assertEquals("ovid", importer.getCLIId());
+    }
+
+    @Test
+    public void testIsRecognizedFormatAccept() throws IOException {
+
+        List<String> list = Arrays.asList("OvidImporterTest1.txt", "OvidImporterTest3.txt", "OvidImporterTest4.txt",
+                "OvidImporterTest5.txt", "OvidImporterTest6.txt", "OvidImporterTest7.txt");
+
+        for (String str : list) {
+            try (InputStream is = OvidImporter.class.getResourceAsStream(str)) {
+                Assert.assertTrue(importer.isRecognizedFormat(is));
+            }
+        }
+    }
+
+    @Test
+    public void testIsRecognizedFormatRejected() throws IOException {
+
+        List<String> list = Arrays.asList("Empty.txt", "OvidImporterTest2.txt");
```

Comment:
Do you want the Empty.txt file removed from the test or do you want it to be a recognized method?
Because the isRecognized() Method just returns true if he finds a String that matches "<[0-9]+>".